zerver/webhooks/slack/view.py

```diff
@@ -0,0 +1,35 @@
+from __future__ import absolute_import
+from django.utils.translation import ugettext as _
+from django.http import HttpRequest, HttpResponse
+from django.utils.translation import ugettext as _
+from zerver.lib.actions import check_send_message, create_stream_if_needed
+from zerver.lib.response import json_success, json_error
+from zerver.lib.validator import check_string, check_int
+from zerver.decorator import REQ, has_request_variables, api_key_only_webhook_view
+from zerver.models import UserProfile, Client
+
+ZULIP_MESSAGE_TEMPLATE = "**{message_sender}** has sent the following message:\n `{text}`"
```

Comment:
I think we probably want a much less verbose heading, e.g. "**{message_sender}**: {text}"